engine/src/main/java/org/terasology/rendering/nui/widgets/UIBox.java

```diff
@@ -57,19 +57,39 @@ public void update(float delta) {
         }
     }
 
+    /**
+     * Gets the widget/layout inside the box.
```

Comment:
The getter/setter descriptions are way too repetitive and the boolean ones ("gets if the contents..." / "sets if the contents...") are just weird.

A better solution is to simply remove the description everywhere:

```java
// getter
/**
 * @return Whether the content is updated at the same time as the box.
 */

// setter
/*
 * @param value Whether the content should be updated at the same time as the box.
 */
```

(If Checkstyle complains about this, make it shut up by setting `allowMissingPropertyJavadoc` to `false` in [our config](https://github.com/MovingBlocks/TeraConfig))